cms/signals/apphook.py

```diff
@@ -1,14 +1,25 @@
 # -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
 import sys
 from django.core.management import color_style
 from django.core.urlresolvers import clear_url_caches
 from django.core.signals import request_finished
 from cms.models import Title
+from cms.utils.apphook_reload import mark_urlconf_as_changed
 
 
 DISPATCH_UID = 'cms-restart'
 
 
+def trigger_server_restart(**kwargs):
```

Comment:
This looks the same as https://github.com/divio/django-cms/pull/4348/files#diff-643105bbf8c6920853343a09e34d38acR38